explainer.md

```diff
@@ -34,26 +34,35 @@ At its core, the API is designed for a website ("verifier") to [transparently](h
 
 Here is an example of how the  the API might be used in practice:
 
+The API needs to be initiated through a user gesture, such as a button click:
+
+```html
+<button onclick="requestLicense()">Request Driver's license<button>
+```
+
+
 ```javascript
-    const digitalCredential = await navigator.identity.get({
-      digital: {
-        providers: [{
-          // Protocol extensibility:
-          protocol: "oid4vp",
-          // An example of an OpenID4VP request to wallets.
-          // Based on https://github.com/openid/OpenID4VP/issues/125
-          request: {
-            nonce: "n-0S6_WzA2Mj",
-            presentation_definition: {
-              // Presentation Exchange request, omitted for brevity
-            }
-          }
-        }],
-      },
-    });
-    // To be decrypted on the server...
-    const encryptedData = digitalCredential.data;
+async function requestLicense() {
+  const oid4pv = {
+    // Protocol extensibility:
+    protocol: "oid4vp", // An example of an OpenID4VP request to wallets. // Based on https://github.com/openid/OpenID4VP/issues/125
```

Comment:
```suggestion
    protocol: "urn:openid.net:oid4vp", // An example of an OpenID4VP request to wallets. // Based on https://github.com/openid/OpenID4VP/issues/125
```

That's the value they are using here: https://github.com/openid/OpenID4VP/pull/155/files

Reply:
I'll see if I can get them to drop that.... using a URN doesn't seem appropriate as otherwise we should parse these as URLs.

Reply:
I thought that we were deliberate in making [`protocol`](https://wicg.github.io/digital-credentials/#the-protocol-member) a `DOMString`, so that we wouldn't have to form an opinion on what these were.

Should we care if this is a URN or not? Or are you suggesting that we reject any `DOMString` that isn't a URL?

Reply:
def should care if it's an URN... because URNs are URLs, and hence it implies URL parsing. We just don't want to get into a world where people are expecting a URL where we are using a DOMString.